Mathlib/Topology/UniformSpace/Equiv.lean

```diff
@@ -135,26 +139,26 @@ protected theorem uniformContinuous (h : α ≃ᵤ β) : UniformContinuous h :=
   h.uniformContinuous_toFun
 #align uniform_equiv.uniform_continuous UniformEquiv.uniformContinuous
 
-@[continuity]
+--@[continuity] -- Porting note: missing attribute
 protected theorem continuous (h : α ≃ᵤ β) : Continuous h :=
-  h.UniformContinuous.Continuous
+  h.uniformContinuous.continuous
 #align uniform_equiv.continuous UniformEquiv.continuous
 
 protected theorem uniformContinuous_symm (h : α ≃ᵤ β) : UniformContinuous h.symm :=
   h.uniformContinuous_invFun
 #align uniform_equiv.uniform_continuous_symm UniformEquiv.uniformContinuous_symm
 
 -- otherwise `by continuity` can't prove continuity of `h.to_equiv.symm`
-@[continuity]
+--@[continuity] -- Porting note: missing attribute
 protected theorem continuous_symm (h : α ≃ᵤ β) : Continuous h.symm :=
-  h.uniformContinuous_symm.Continuous
+  h.uniformContinuous_symm.continuous
 #align uniform_equiv.continuous_symm UniformEquiv.continuous_symm
 
 /-- A uniform isomorphism as a homeomorphism. -/
-@[simps]
+-- @[simps] -- Porting note: restore `simps`
```

Comment:
What is the issue with simps? If it can't be fixed easily, I guess we should manually add the simp-lemmas it generated as a tmp workaround.